docs/tutorials/hello-world.md

```diff
@@ -126,18 +124,18 @@ import { toByteString, sha256 } from 'scrypt-ts'
     const deployTx = await instance.deploy(42)
     console.log('Helloworld contract deployed: ', deployTx.id)
 
-    // contract call
+    // call the contract
     const { tx: callTx } = await instance.methods.unlock(message)
     console.log('Helloworld contract `unlock` called: ', callTx.id)
-
 })()
 ```
 
-Run the following command:
+Run the following command to deploy AND call our example contract.
```

Comment:
for the tutorial, deploying and calling make sense.

but in practice, deploy and calling a likely separate, so i wanted to call this out